client/src/components/JurisdictionAdmin/BatchRoundDataEntry/BatchRoundDataEntry.tsx

```diff
@@ -0,0 +1,160 @@
+  jurisdictionId: string
+  roundId: string
+}
+
+const BatchRoundDataEntry: React.FC<IProps> = ({
+  electionId,
+  jurisdictionId,
+  roundId,
+}) => {
+  const batchesQuery = useBatches(electionId, jurisdictionId, roundId)
+  const contestsQuery = useContestsJurisdictionAdmin(electionId, jurisdictionId)
+  const recordBatchResults = useRecordBatchResults(
+    electionId,
+    jurisdictionId,
+    roundId
+  )
+  const { confirm, confirmProps } = useConfirm()
+
+  const [searchQuery, setSearchQuery] = useState('')
+  const [debouncedSearchQuery] = useDebounce(searchQuery)
+  const [selectedBatchId, setSelectedBatchId] = useState<IBatch['id']>()
+  const [areChangesUnsaved, setAreChangesUnsaved] = useState(false)
+
+  const batches = batchesQuery.isSuccess ? batchesQuery.data.batches : []
+  const filteredBatches = batches.filter(batch =>
+    batch.name.toLowerCase().includes(debouncedSearchQuery.toLowerCase())
+  )
+  const selectedBatch = batches.find(batch => batch.id === selectedBatchId)
+
+  // Auto-select first batch on initial load
+  useEffect(() => {
+    if (!selectedBatchId && filteredBatches.length > 0) {
+      setSelectedBatchId(filteredBatches[0].id)
+    }
+  }, [filteredBatches, selectedBatchId, setSelectedBatchId])
+
+  // Auto-select first search match
+  useEffect(() => {
+    if (
+      debouncedSearchQuery &&
+      filteredBatches.length > 0 &&
+      !areChangesUnsaved
+    ) {
+      setSelectedBatchId(filteredBatches[0].id)
+    }
+  }, [
+    debouncedSearchQuery,
+    filteredBatches,
+    areChangesUnsaved,
+    setSelectedBatchId,
+  ])
+
+  if (!batchesQuery.isSuccess || !contestsQuery.isSuccess) {
+    return null
+  }
+
+  const areResultsFinalized = Boolean(batchesQuery.data.resultsFinalizedAt)
+  // Batch comparison audits only support a single contest
+  const [contest] = contestsQuery.data
+
+  const selectBatch = (batchId: string) => {
+    if (areChangesUnsaved) {
+      confirm({
+        title: 'Unsaved Changes',
+        description:
+          'You have unsaved changes. ' +
+          'Are you sure you want to leave this batch without saving changes?',
+        yesButtonLabel: 'Discard Changes',
+        yesButtonIntent: 'danger',
+        onYesClick: () => setSelectedBatchId(batchId),
+        noButtonLabel: 'Cancel',
+      })
+      return
+    }
+    setSelectedBatchId(batchId)
+  }
+
+  return (
+    <Container>
+      {areResultsFinalized && (
+        <Callout icon="tick-circle" intent="success">
+          Tallies finalized
+        </Callout>
+      )}
+      <ListAndDetail>
+        <List
+          search={{
+            placeholder: 'Search batches...',
+            setQuery: setSearchQuery,
+          }}
+        >
+          {filteredBatches.map(batch => (
+            <ListItem
+              key={batch.id}
+              onClick={() => selectBatch(batch.id)}
+              selected={batch.id === selectedBatchId}
+            >
+              {batch.name}
+            </ListItem>
```

Comment:
I just realized there's a task that's quite difficult in this new view: finding a batch with no tallies entered (e.g. if you want to finalize tallies, but you missed one batch). I'm not sure that we need to fix it now, but adding a simple tick mark icon to the right end of the list item might be an easy fix.

Reply:
Voila! https://github.com/votingworks/arlo/pull/1661/commits/a25fd1e3f0a221670c839a6e9602e26a636f2aa5

<img width="255" alt="checks" src="https://user-images.githubusercontent.com/12616928/199326525-9f1547a6-338e-4031-b6c2-5f1d908e2b7f.png">

Reply:
That looks great!